jest.config.ts

```diff
@@ -12,8 +10,13 @@ const config: InitialOptionsTsJest = {
 
   collectCoverage: true,
   coverageDirectory: 'coverage',
-  coverageReporters: ['json', 'lcov', 'text'],
+  coverageReporters: ['lcov'],
```

Comment:
@florkbr I think we only need the `lcov` reporter for integration with the [CodeCov](https://docs.codecov.com/docs) service.

Side note, `patternfly-react` also uses just the `lcov` reporter, see [here](https://github.com/patternfly/patternfly-react/blob/main/jest.config.js).

Reply:
We may eventually need the other reporters, but I think this is fine for now.